Daf.Meta.Editor/ViewModels/MainViewModel.cs

```diff
@@ -387,9 +424,12 @@ private void DeleteDataSource()
 
 				if (result == MessageBoxResult.Yes)
 				{
+					// TODO: This points to the wrong path. Can not currently delete DataSources.
```

Comment:
Should it be pointed at the correct path?

Reply:
Possibly. It should be an easy fix but we talked about not expanding the scope of this PR and the same issue is present in Master. Would you like to see it fixed in this PR?